README.md

```diff
@@ -28,32 +31,80 @@ The decision to fork the original mender-mcu-client was made to:
 * **Provide Official Support**: Ensure that the project receives the necessary attention and
   resources from Northern.tech to meet the needs of the community and enterprise users.
 
+## Getting started
```

Comment:
We should keep "Get started". We use to have "getting" in docs.mender.io and we switched it and we want to be consistent